primitives/chains/kusama/src/lib.rs

```diff
@@ -0,0 +1,117 @@
+// Copyright 2019-2020 Parity Technologies (UK) Ltd.
```

Comment:
```suggestion
// Copyright 2019-2021 Parity Technologies (UK) Ltd.
```

Reply:
Oh I just realized these aren't actually new files, they've been in the repo since last year (for Kusama and Polkadot anyways). I don't think the diff should be showing this as a new file - did you use `git mv` when making these new crates?

Reply:
`git mv` doesn't matter. Git tracks moves by comparing the content of the files, since this file has been moved, but then significantly changed `git` is not able to keep track of this.